src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,96 @@
+    ```
+
+1. Deploy the gateway:
```

Comment:
```suggestion
1. Deploy the gateway contract. If you have customized/built your own gateway, then you can deploy that. You can also instantiate the [existing](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json#L409) gateway contract which has code id `7` instead of deploying a new one:
```
some folks have their own custom gateway and shouldn't deploy ours. otherwise they can instantiate directly instead of building their own